### DISCUSSION TITLE
I would like to ask if the edc connector is compatible with Testsuite.postman_collection in IDSA ?

### DISCUSSION THREAD

**Discussion by sasfc** [eclipse-edc/Connector#4663] [Q&A] [answered]
The Testsuite.postman_collection is https://github.com/International-Data-Spaces-Association/IDS-testbed/blob/master/Testsuite/Testsuite.postman_collection.json

**Comment by jimmarino** [comment, marked as answer]
Hi,

You should ask them, as that repository is not an EDC one. My guess is "no," since EDC implements [DSP](https://github.com/eclipse-dataspace-protocol-base/DataspaceProtocol), not the old IDSA protocol, which is something different.

**Reply by sasfc** [reply]
Thanks for answering. So did EDC have an old version that implemented IDSA protocol ?

**Reply by paullatzelsperger** [reply]
no, nothing that is even remotely supported. what would be the point of that. however, you may be able to cobble together something from the git reflog from before we switched to DSP, but you are on your own there.

**Reply by sasfc** [reply]
Thanks

**Reply by FantZero** [reply]
> Hi, 
> 
> You should ask them, as that repository is not an EDC one. My guess is "no," since EDC implements [DSP](https://github.com/eclipse-dataspace-protocol-base/DataspaceProtocol), not the old IDSA protocol, which is something different.

Why do you choose to use DSP protocol instead of IDSA? Would you please explain.

**Reply by jimmarino** [reply]
This has been documented in numerous places. DSP is a distinct specification that succeeds the old IDSA protocols under the governance of the Eclipse Foundation.

**Reply by FantZero** [reply]
> This has been documented in numerous places. DSP is a distinct specification that succeeds the old IDSA protocols under the governance of the Eclipse Foundation.

thanks a lot, I want to pass IDS certification, any suggestions? Or is there a reference in the EDC open source project?

**Reply by jimmarino** [reply]
This is an open-source project, not a certification effort. Please contact IDSA, not this board.
